packages/documentation/src/stories/components/forms/input/input.stories.ts

```diff
@@ -98,11 +99,7 @@ const meta: Meta = {
     },
     size: {
       name: 'Size',
-      description: "Sets the size of the component's appearance.",
-      if: {
```

Comment:
This condition still seems necessary: the sizes do not apply to floating labelled input.

Reply:
Good catch, it will be part of the next PR